src/main/java/eu/dissco/orchestration/backend/controller/DataMappingController.java

```diff
@@ -55,10 +58,10 @@ public ResponseEntity<JsonApiWrapper> createDataMapping(Authentication authentic
   public ResponseEntity<JsonApiWrapper> updateDataMapping(Authentication authentication,
       @PathVariable("prefix") String prefix, @PathVariable("suffix") String suffix,
       @RequestBody JsonApiRequestWrapper requestBody, HttpServletRequest servletRequest)
-      throws JsonProcessingException, NotFoundException, ProcessingFailedException {
+      throws JsonProcessingException, NotFoundException, ProcessingFailedException, ForbiddenException {
     var dataMapping = getDataMappingRequestFromRequest(requestBody);
     var id = prefix + '/' + suffix;
-    var userId = authentication.getName();
+    var userId = getAgent(authentication).getId();
     log.info("Received update request for data mapping: {} from user: {}", dataMapping, userId);
     String path = appProperties.getBaseUrl() + servletRequest.getRequestURI();
     var result = service.updateDataMapping(id, dataMapping, userId, path);
```

Comment:
Shouldn't the full agent go to the `service.updateMapping()`? Maybe for another task as it would be nice to have both the id and name in the CUT event

Reply:
https://naturalis.atlassian.net/browse/DD-1382